### PR TITLE
Some simple optimizations

### PR DESCRIPTION
**Pull request by cyyever**

### CODE REVIEW

**Comment by peterjc123** [comment]
@cyyever Could you please fix the formatting issues?

**Review by VitalyFedyunin — APPROVED** [review]

**Comment by cyyever** [comment]
> @cyyever Could you please fix the formatting issues?
Fixed.

File: torch/csrc/jit/runtime/logging.h [hunk 1]
```diff
@@ -61,7 +61,7 @@ struct JITTimePoint {
 };
 
 TORCH_API JITTimePoint timePoint();
-TORCH_API void recordDurationSince(const std::string& name, JITTimePoint tp);
+TORCH_API void recordDurationSince(const std::string& name, const JITTimePoint& tp);
```
Comment (peterjc123) [inline torch/csrc/jit/runtime/logging.h:64]: clang-format thinks that this line is too long.
```suggestion
TORCH_API void recordDurationSince(const std::string& name,
                                   const JITTimePoint& tp);
```

**Comment by VitalyFedyunin** [comment]
clang test still fails

**Comment by cyyever** [comment]
> clang test still fails

Fixed

**Comment by VitalyFedyunin** [comment]
Can you please rebase, the amount of merge conflicts in tests logs is suspicious.

**Pull request closed without merging**